### ISSUE TITLE
Uncaught AttributeError in sendchat.py line 30

### ISSUE THREAD

**Issue by pszemraj** [paul-gauthier/aider#1658]
Aider version: 0.54.12
Python version: 3.10.14
Platform: Linux-6.8.0-40-generic-x86_64-with-glibc2.35
Python implementation: CPython
Virtual environment: No
OS: Linux 6.8.0-40-generic (64bit)
Git version: git version 2.34.1

An uncaught exception occurred:

```
Traceback (most recent call last):
  File "aider", line 8, in <module>
    sys.exit(main())
  File "main.py", line 682, in main
    if args.message:
  File "base_coder.py", line 730, in run
    def get_input(self):
  File "base_coder.py", line 773, in run_one
    return
  File "base_coder.py", line 1183, in send_message
    content += "\n^C KeyboardInterrupt"
  File "base_coder.py", line 1862, in auto_commit
    context += "\n" + msg["role"].upper() + ": " + msg["content"] + "\n"
  File "repo.py", line 106, in commit
    elif self.attribute_commit_message_committer:
  File "repo.py", line 186, in get_commit_message
    continue
  File "sendchat.py", line 38, in wrapper
    retry_exceptions(),
  File "sendchat.py", line 30, in retry_exceptions
    litellm.llms.anthropic.chat.AnthropicError,
AttributeError: module 'litellm.llms.anthropic' has no attribute 'AnthropicError'

```

**Comment by fry69** [comment]
Thank you for filing this issue.

The fix is in the latest version (`0.57.1`), please update aider with
```
aider --upgrade
```

**Comment by pszemraj** [comment]
thanks!
